tests/testthat/test-module_teal.R

```diff
@@ -65,55 +65,6 @@ transform_list <<- list(
   )
 )
 
-testthat::describe("srv_teal lockfile", {
```

Comment:
Why are you removing this on the PR?

If it's giving you problems locally, try installing `renv@1.0.11` that should have a fix

Reply:
Thanks for this. I was sloppy here. Newer `renv` helps also! 👍